srcpkgs/oblibs/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'oblibs'
+pkgname=oblibs
+version=0.0.1.1
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=gnu-configure
+makedepends="skalibs-devel"
+short_desc="Obarun library"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="ISC"
+homepage="https://framagit.org/Obarun/oblibs"
+distfiles="${homepage}/-/archive/v${version}/${pkgname}-v${version}.tar.bz2"
+checksum=fbb0eaf9223805124f76c086a2d1a444649d077ef86bc05762776b8c0cfad1b3
+
```

Comment:
I think i found the error.

on x86_64-musl

libdir := /usr/lib/obslibs

on i686

libdir := /usr/lib32



On gnu-configure we pass --libdir=/usr/lib32. but oblibs is not gnu-configure, use the `configure` build_style like in `skalibs`